config/default/field.field.paragraph.civictheme_message.field_c_p_message_type.yml

```
@@ -0,0 +1,23 @@
+field_name: field_c_p_message_type
+entity_type: paragraph
+bundle: civictheme_message
+label: Type
+description: ''
+required: true
+translatable: false
+default_value:
+  -
+    value: information
+default_value_callback: ''
+settings: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)